finger-exercises-react/react/tic-tac-toe/src/app/screens/PrevGames/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Spinner from 'react-spinkit';
+
+import { getMatchesAction } from '../../../redux/PrevGames/actions';
+
+import styles from './styles.scss';
+
+const winner = game => {
+  if (game.winner === 'tie') {
+    return 'Empate';
+  }
+  return game[game.winner];
+};
+
+class PrevGames extends React.Component {
+  componentDidMount() {
+    this.props.getMatches();
+  }
+
+  renderGames() {
+    if (this.props.error) {
+      return <p> Error API Request</p>;
```

Comment:
To be consistent show a message in Spanish, and something a user would understand 😄